src/test/java/uk/gov/hmcts/cmc/claimstore/services/ccd/CoreCaseDataServiceFailureTest.java

```diff
@@ -564,3 +564,4 @@ public void savePaidInFullSubmitEventFailure() {
         service.savePaidInFull(claim.getId(), paidInFull, AUTHORISATION);
     }
 }
+ClaimDocumentMapperTest.java:
```

Comment:
what is this?

Reply:
My bad. I can type faster than Intellij can respond :-)

I actually fixed and pushed about an ago. Sorry about.